Adding Burstable family

macieksarnowicz opened the pull request:
Updated the quick-start managed services to add a blurb about new VM families. Also updated the screenshot to show the family selector on the VM Create page.

ozgune commented:
Hi @macieksarnowicz - thank you for the pull request. I wanted to drop a high level comment.

* I retract my comments that relates to Burstable VMs and Burstable PostgreSQL. Your update brings us closer to how AWS communicates burstable instances. Please feel free to resolve my first comment.
* Given that our definition of burstables is different than standard instances, let's **not** update the ParadeDB section. I don't think it's worth adding a new section on burstables either.

Once you remove the update to ParadeDB, please feel free to merge this pull request.

macieksarnowicz commented:
> Hi @macieksarnowicz - thank you for the pull request. I wanted to drop a high level comment.
> 
> * I retract my comments that relates to Burstable VMs and Burstable PostgreSQL. Your update brings us closer to how AWS communicates burstable instances. Please feel free to resolve my first comment.
> * Given that our definition of burstables is different than standard instances, let's **not** update the ParadeDB section. I don't think it's worth adding a new section on burstables either.
> 
> Once you remove the update to ParadeDB, please feel free to merge this pull request.

I removed the changes in ParadeDB and rebased the PR on the lastest. This is ready for merge. @pykello - could you please help merge it?

ozgune reviewed (commented):
Looks good to me.
